.make/Makefile.build

```diff
@@ -4,13 +4,19 @@ clean: ## Removes binary, cache folder and docker images
 	@rm -rf $(PLUGIN_DEPLOYMENTS_DIR)
 
 .PHONY: tools
-tools: ## Installs required go tools
+tools: tools.timestamp
+
+tools.timestamp: ## Installs required go tools
 	@go get -u github.com/alecthomas/gometalinter && gometalinter --install
 	@go get -u github.com/onsi/ginkgo/ginkgo
 	@go get -u github.com/onsi/gomega
 	@go get -u golang.org/x/tools/cmd/goimports
 	@go get -u github.com/jteeuwen/go-bindata/...
+	@touch tools.timestamp
```

Comment:
What's the purpose for that file?

Reply:
I intended to store the timestamp and then run `dep` on these as is done for the other `fabric8` repos  - https://github.com/fabric8-services/fabric8-jenkins-proxy/blob/master/Makefile#L66

Reply:
Can you explain how this file is used? I might be missing something.

Reply:
Sorry, my bad. The purpose of the file is to prevent people from installing the same tools again and again and waste bandwidth by preventing this command if it was run before and we don't need to run it again.

Reply:
I think we can vote here if we really need this file, so @dipak-pawar @hemanik @lordofthejars @MatousJobanek please :+1: or :-1: this comment.

If we :+1:, then we have to add following:

- add this file to `.gitignore` so we won't accidentally commit it
- probably have a cleanup target